jobs/download_cubes.js

```diff
@@ -1,63 +1,103 @@
+/* eslint-disable no-await-in-loop */
 // run with: node --max-old-space-size=8192 populate_analytics.js
 // will oom without the added tag
 
 // Load Environment Variables
 require('dotenv').config();
-const fs = require('fs');
-
-const path = (batch) => `jobs/export/cubes/${batch}.json`;
 
 const mongoose = require('mongoose');
 
+const AWS = require('aws-sdk');
 const Cube = require('../models/cube');
 const carddb = require('../serverjs/cards.js');
-const AWS = require('aws-sdk');
+
 const s3 = new AWS.S3({
   accessKeyId: process.env.AWS_ACCESS_KEY,
-  secretAccessKey: process.env.AWS_SECRET_ACCESS_KEY
+  secretAccessKey: process.env.AWS_SECRET_ACCESS_KEY,
 });
 
 const batchSize = 1000;
 
 const processCube = (cube) => {
-  return cube.cards.map((card) => carddb.cardFromId(card.cardID).name_lower);
+  const {
+    date_updated,
+    numDecks,
+    type,
+    name,
+    owner,
+    owner_name,
+    tags,
+    image_uri,
+    image_artist,
+    image_name,
+    card_count,
+    users_following,
+    isListed,
```

Comment:
I'm not sure what else these exports are used for, but we should maybe consider just not dumping the cubes that are unlisted.